src/env/env.rs

```diff
@@ -40,6 +44,19 @@ fn print_env(null: bool) {
     }
 }
 
+#[cfg(not(windows))]
+fn build_command( mut args: Vec<String> ) -> ( String, Vec<String> ) {
+    let command = args[0].clone();
```

Comment:
This can actually just be `let command = args.remove(0);`, which will save an allocation.

Reply:
Will do...

Reply:
Done.